src/main/resources/fabric.mod.json

```diff
@@ -3,7 +3,7 @@
     "id": "tis3d",
     "name": "TIS-3D",
     "description": "TIS-100 inspired low-tech computing for Minecraft.",
-    "version": "1.5.0",
+    "version": "1.5.99",
```

Comment:
Sure, the 99 was just for me to differentiate my WIP code, but I should have done that via `BUILD_NUMBER` instead of abusing the mod version. If the MC version is the super-major, that means the breaking changes in `api` are also covered by it, right?

Reply:
Yes, those changes would be covered by that; pretty sure there's precedent for that, too.